src/sgp/Types.h

```diff
@@ -54,11 +54,6 @@ typedef char            CHAR8;
 typedef unsigned char		BOOLEAN;
 typedef void *					PTR;
 typedef UINT8						BYTE;
-typedef CHAR8						STRING512[512];
-
-#define SGPFILENAME_LEN 100
-typedef CHAR8 SGPFILENAME[SGPFILENAME_LEN];
-
 
 #ifndef TRUE
 #define TRUE 1
```

Comment:
While we have to recompile everything because you're touching Types.h, BAD_INDEX is also unused.